### PR TITLE
LNBits version bump (0.8.0)

### PR DESCRIPTION
**Pull request by kristapsk**
#### What

Bump LNBits version to 0.8.0.

### Why

[New version was released month ago](https://github.com/lnbits/lnbits-legend/releases/tag/0.8.0).

#### Scope

- [ ] significant change to core configuration
- [X] independent bonus guide
- [ ] simple bug fix

### CODE REVIEW

File: guide/bonus/lightning/lnbits.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Table of contents
 * Download the source code directly from GitHub, create a virtual environment, and install all dependencies with pip.
 
   ```sh
-  $ git clone --branch 0.7.0 https://github.com/lnbits/lnbits-legend lnbits
+  $ git clone --branch 0.8.0 https://github.com/lnbits/lnbits-legend lnbits
```
Comment (VajraOfIndra) [inline guide/bonus/lightning/lnbits.md:109]:
```suggestion
  $ git clone https://github.com/lnbits/lnbits-legend lnbits
```
Is declaring the branch necessary for this program? If we could remove this option, there would be no need to keep updating this section in future releases.

File: guide/bonus/lightning/lnbits.md [hunk 2]
```diff
@@ -251,14 +251,14 @@ Updating to a [new release](https://github.com/lnbits/lnbits-legend/releases){:t
...
-* Fetch the latest GitHub repository information, display the release tags (use the latest `0.7.0` in this example), and update:
+* Fetch the latest GitHub repository information, display the release tags (use the latest `0.8.0` in this example), and update:
 
   ```sh
   $ cd /home/lnbits/lnbits
   $ git fetch
   $ git reset --hard HEAD
   $ git tag
-  $ git checkout 0.7.0
+  $ git checkout 0.8.0
```
Comment (VajraOfIndra) [inline guide/bonus/lightning/lnbits.md:261]: I tried to use the command you used for other programs in your recent PRs.. I now undersatnd why you didn't use for LNBits! :)

**Review by VajraOfIndra — APPROVED** [review]
:+1: thanks for this update @kristapsk 

tACK

Just a quick question regarding the need for the --branch option, see comment below.

File: guide/bonus/lightning/lnbits.md (hunk 1 above)
Comment (kristapsk) [reply guide/bonus/lightning/lnbits.md:109]: I guess not. Was just installing LNBits on a different machine (not even ARM-based, but RaspiBolt guides are starting to be helpful for time saving even on x86 VPSes) and noticed new release. Just replaced 0.7.0 with 0.8.0 everywhere and checked that commands work.

Comment (VajraOfIndra) [reply guide/bonus/lightning/lnbits.md:109]: Ok, I'll merge as is but maybe we could consider removing this option in the future to quicken the maintenance process.

**Pull request merged**